libraries/scopes/src/satisfaction.js

```diff
@@ -1,13 +1,4 @@
-const assert = require('assert');
-const {validateScopeSets} = require('./validate');
-
-const validateScopePatterns = (scopePatterns) => {
-  assert(scopePatterns instanceof Array && scopePatterns.every((scope) => {
-    return typeof scope === 'string';
-  }), 'scopes must be an array of strings');
-};
-
-const patternMatch = exports.patternMatch = (pattern, scope) => {
+exports.patternMatch = (pattern, scope) => {
```

Comment:
should we just rename this file at this point if it only has `patternMatch` in it?

Reply:
Yeah, honestly most of the code could fit in one file at this point..